### ISSUE TITLE
What to do about 64x128 Resolution?

### ISSUE THREAD

**Issue by fourstix** [fourstix/1802PixieVideoTTY#1]
Currently I have no way to test 64x128 Resolution.  I removed the 64x128 logic from the code source files I used.
It might be a better approach to restore deleted 64x128 logic to original graphics assembly source files so they match the originals.  And then add commented out conditional assembly logic for 64x128 resolution with comments noting that the code has not been tested.

The logic for 64x128 should follow the 64x64 and 64x32 logic, except the  y values are twice those for 64x64 and four times those for 64x32.  But I'm uncomfortable to release code that I cannot test.

**Comment by fourstix** [comment]
Well I coded it, but I wasn't very happy with not being able to test the results.  I can't really tell if what I coded worked or not.

Maybe the underlying graphics code isn't quite what I expected at 64x128 or maybe there is an issue with the code I wrote.  I can't really debug anything having only 64x64 resolution available.   So I am going to close this issue.

**Comment by fourstix** [comment]
I figured out a way to debug 64x128.  The Pixie Video simulator will sample every other pixel row at 64x64, so 64x128 will show up mangled, but at least one can count rows.  I created the code update and tested it as best I can at 64x128 resolution.

I did find and fix one issue with the underlying DrawSprite routine in Graphics1861.asm.  The routine wasn't calculating the y offset correctly for 64x128 resolution.  I updated the routine with the same logic approach used to calculate the y offset value for 64x128 resolution.
